### DISCUSSION TITLE
Now playing info/ question

### DISCUSSION THREAD

**Discussion by Robdewild** [AzuraCast/AzuraCast#4835] [General]
Hello azuracast users,

i have a medialibrary where the songtiltel contains title, and artist in de id3 tag „title“
Now playing shows the artist 2 times .
Example : Bon Jovi - Bon Jovi - bad medicine.
Is there a way to edit liquidsoap script to show only title of the track, and forget the artist field in id3 tag and show proper now playing info?
I can rewrite all the tags by blanking out the „artist“ field. But perhaps this can be done easier…
Not a big issue, but a bit annoying 😜
Thanks for the ideas in advance.

grtz
Rob

**Comment by Robdewild** [comment]
![99EBAB79-6688-4D28-A94C-6C7EBBD78841](https://user-images.githubusercontent.com/88682313/144659713-f211523d-1444-4119-8cca-aa0a55a2063c.png)

**Comment by Robdewild** [comment]
Thats how it looks like…

**Reply by SpeedyCharly** [reply]
Is that really such a problem?  Your easiest option is to edit the id3 tags in your media files, leaving only the title in Title and the artist in Contributing Artist and your player will look like this;
![Image4](https://user-images.githubusercontent.com/66107794/145147687-6883160d-4e7c-40dc-94bb-7e8202f9b294.jpg)

Minus my customization that is :P

**Comment by Robdewild** [comment]
Yes, that is correct.
I know i can change the id3 tags, but the question was if there was another way to do this without those changes.


via 
Mobiel

**Reply by SpeedyCharly** [reply]
> Yes, that is correct. I know i can change the id3 tags, but the question was if there was another way to do this without those changes. via Mobiel

Everyone is asking Azuracast to be a do-everything Swiss army knife of an app.  The thing to remember is that while a Swiss army knife does everything, it is great at nothing...

**Comment by bugoverfl0w** [comment]
Cool, you ask and you answer

**Comment by Robdewild** [comment]
Sorry guys,
It was a question only.
Didn‘t know that a few people have a short fuse.
Already mentioned that it‘s not a large issue.
Sorry for any inconvenience.

Grtz,
Rob

via 
Mobiel

**Reply by SpeedyCharly** [reply]
Not short fused; merely stating fact.  I apologize if you took this as a personal attack, such wasn't the intent...

**Comment by Robdewild** [comment]
Solved:
settings.encoder.metadata.export.set(["title"])
